buildSrc/build.gradle.kts

```diff
@@ -4,11 +4,15 @@ plugins {
 
 repositories {
     jcenter()
+    maven {
+        url = uri("https://plugins.gradle.org/m2/")
```

Comment:
Isn't `gradlePluginPortal` just a shortcut for this url?

Reply:
Fixed